setup.py

```diff
@@ -5,7 +5,7 @@
 
 # Add that to the dev reqs file for now, since it's not needed when running
 dependencies = ['click', 'genologics', 'requests-cache', 'pyyaml', 'nose', 'PyPDF2',
-                'lxml']
+                'lxml', 'coverage', 'pep8radius']
 
```

Comment:
Yes, for now, they're listed in setup.py. We'll probably want to use a requirements file in the future, so people don't have to install dev requirements in production, e.g nose, coverage and pep8radius, but I'll clean that up later.

(The other repo (that uses this) is however setup with a conda metadata file on the prod servers.)